lerna-package-lock.json

```diff
@@ -1975,7 +1975,7 @@
 			}
 		},
 		"@fluid-experimental/property-binder-previous": {
-			"version": "npm:@fluid-experimental/property-binder-previous@2.0.0-internal.2.0.0",
+			"version": "npm:@fluid-experimental/property-binder@2.0.0-internal.2.0.0",
```

Comment:
I'm not sure whats happening with all these changes. They seem unrelated to the PR though, so ideally, they shouldn't be included here. If they unavoidably happen, even on main, when doing an npm install (after deleting node_modules), then you could either revert these changes, or commit them in a different PR.

Reply:
This may be an issue with the type test generation itself. I'll investigate later today.